project-canvas/src/components/backlog-view/Item.tsx

```diff
@@ -1,32 +1,128 @@
 import { Draggable } from "react-beautiful-dnd"
 
-export function Item({ issueKey, summary, index, status, columnId }: Issue) {
+export function Item({
+  issueKey,
+  summary,
+  index,
+  status,
+  type,
+  storyPoints,
+}: Issue) {
+  let icon: JSX.Element
+  let iconGradient1: string
+  let iconGradient2: string
+  switch (type) {
+    case "Story":
+      icon = <IconBookmark />
+      iconGradient1 = "teal"
+      iconGradient2 = "lime"
+      break
+    case "Task":
+      icon = <IconCheck />
+      iconGradient1 = "teal"
+      iconGradient2 = "blue"
+      break
+    case "Bug":
+      icon = <IconBug />
+      iconGradient1 = "orange"
+      iconGradient2 = "red"
+      break
+
+    default:
+  }
+
   return (
     <Draggable draggableId={issueKey} index={index}>
       {(provided) => (
         <Box
           sx={{
-            backgroundColor: "#eee",
-            borderRadius: 4,
-            padding: "4px 8px",
+            backgroundColor: "white",
+            borderRadius: 2,
+            padding: "5px",
             transition: "background-color .8s ease-out",
-            marginTop: 8,
+            margin: 8,
+            boxShadow:
+              "0 0 1px 0 rgb(9 30 66 / 31%), 0 2px 4px -1px rgb(9 30 66 / 25%)",
 
             ":hover": {
-              backgroundColor: "#fff",
+              backgroundColor: "#ebecf0",
               transition: "background-color .1s ease-in",
             },
           }}
           ref={provided.innerRef}
           {...provided.draggableProps}
           {...provided.dragHandleProps}
         >
-          <Text>{summary}</Text>
-          <Text>{status}</Text>
-          <Text>{issueKey}</Text>
-          <Text>{columnId}</Text>
+          <Stack sx={{ gap: "0px" }}>
+            <Box
+              sx={{
+                display: "flex",
+                alignItems: "center",
+                height: "22px",
+                margin: "5px",
+              }}
+            >
+              <ThemeIcon
+                size="lg"
+                variant="gradient"
+                gradient={{ from: iconGradient1, to: iconGradient2, deg: 105 }}
+                sx={{
+                  width: "16px",
+                  height: "16px",
+                  minHeight: "16px",
+                  minWidth: "16px",
+                }}
+              >
+                {icon}
+              </ThemeIcon>
+              <Text
+                color="blue"
+                ml={5}
+                sx={{
+                  ":hover": {
+                    textDecoration: "underline",
+                    cursor: "pointer",
+                  },
+                }}
+              >
+                {issueKey}
+              </Text>
+              <Text ml={10} sx={{ flex: "1" }}>
+                {summary}
+              </Text>
+              {storyPoints && (
+                <Text
+                  sx={{
+                    height: "1.2em",
+                    width: "2.5em",
```

Comment:
use the properties `h` and `w`

Reply:
an example of how can i specify the unis em

Reply:
`h="1.2em"`